10up/classifai
# fix/561: delay image generation when using the Media Inserter

Closes #561



### Description of the Change


Adds a modified version of debounce to add delay to an already debounced callback.



### How to test the Change


1. Pull the fix branch
2. To really see the difference, increase the delay from `700` to `2000` in this line `fetch: debounce( imageFetcher, 700 ),`
3. In the media inserter, type very slowly and observe that there are no multiple AJAX calls

### Changelog Entry


> Fixed - Delay image generation when using the Media Inserter


### Credits

Props @dkotter @Sidsector9 


### Checklist:
